apps/wallet-dashboard/components/transactions/TransactionTile.tsx

```diff
@@ -51,22 +52,29 @@ export function TransactionTile({ transaction }: TransactionTileProps): JSX.Elem
 
     const balanceChanges = transactionSummary?.balanceChanges;
 
-    function getAmount(tx: ExtendedTransaction) {
-        if ((isTimelockedStaking || isTimelockedUnstaking) && tx.raw.events) {
-            return getTransactionAmountForTimelocked(
-                tx.raw.events,
+    const [balance, coinType] = (() => {
+        if ((isTimelockedStaking || isTimelockedUnstaking) && transaction.raw.events) {
```

Comment:
If the tx if of type migration we should also always render the iota balance balance